src/view/ChewingEditor.cpp

```diff
@@ -128,6 +129,63 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString translatedCaption = tr("%1%2")
+        .arg(tr("<h3>About Chewing Editor</h3>"))
+        .arg(tr("<p>Version %1 (%2).</p>")
+            .arg(PROJECT_VERSION)
+            .arg(PROJECT_GIT_VERSION));
+    QString translatedText = tr(
+        "<p>Chewing editor is a cross platform Chewing user phrase editor. "
+        "It provides a easy way to manage user phrase. With it, user can "
+        "customize their user phrase to increase input performance.</p>"
+
+        "<p>Chewing editor licensed under the GNU GPL version 2.1 or later version.</p>"
```

Comment:
Chewing editor **is** licensed.
and the https://github.com/chewing/chewing-editor says it's under **2.0** version

ps. (not toward this PR) I really don't like the ambiguous `or later version`. GPL2 and GPL3 is quite different...